docs/metrics/index.md

```diff
@@ -0,0 +1,150 @@
+([1](https://github.com/meandavejustice/min-vid/blob/master/docs/metrics.md),
+[2](https://github.com/mozilla/universal-search/blob/master/docs/metrics.md),
+[3](https://github.com/internetarchive/FirefoxNoMore404s/blob/master/docs/metrics.md))
+but it's important to understand that this is an evolving process and these
+examples may not represent the current needs (for example, none of the above
+leverage Google Analytics).
+
+The Metrics plan should be looked at by the Mozilla Measurement & Analytics
+Teams to make sure avoid implementation troubles.
+
+# Implementing Metrics in an Experiment
+
+The Metrics Plan you've written should serve as a blueprint for implementation.
+
+## Telemetry
+To use Telemetry you'll need to make sure the Mozilla Measurement Team is aware
+a new experiment is starting up.  That involves filing a bug to get Test Pilot
+Experiment data into re:dash, and then creating a re:dash dashboard.  Feel free
+to [clone these two bugs](https://bugzilla.mozilla.org/buglist.cgi?quicksearch=1297199%2C+1297200).
+
+The Measurement Team will be looking at the Metrics Plan you wrote to
+understand what is needed so it's important that it includes the schema you'll
+be logging your data to.
+
+## Google Analytics
+To use Google Analytics you'll need to have your Google Analytics credentials
+to record data.  Using your own account here instead of a generic Test Pilot
+account makes the transition easy when your experiment graduates from the Test
+Pilot program.  (Note that for the extent of the Test Pilot experiment you'll
+need to grant access to analytics for the Test Pilot team).
+
+In order to take full advantage of Google Analytics it's important to follow
+the Metrics Plan you outlined above -- this will let you use the built-in
+e-commerce, funnel, and goal conversion features which will lead to speedy
+analysis.
+
+## Sending Data
+
+The actual function to report event based data is called `testpilot.sendEvent()` and is
+provided by the `testpilot-metrics` node module. To use it:
+
+Add `testpilot-metrics` to your package.json file
+```javascript
+"dependencies": {
+  "testpilot-metrics": "1.0.0",
+  ...
+}
+```
+
+Set up your Google Analytics credentials and enable your endpoints
+```javascript
+TODO: code here
+```
+
+Call testpilot.sendEvent()
+```javascript
+TODO: code here
+```
+
+The testpilot.sendEvent() function accepts these parameters:
+
+* `event`: What is happening?  eg. `click`
+* `object`: What is being affected?  eg. `home-button-1`
+* `category` (optional): If you want to add a category for easy reporting
+  later. eg. `mainmenu`
+* `variant` (optional): An identifying string if you're running different
+  variants. eg. `cohort-A`
+
+These paramters will be combined with other data (like a timestamp and system
+information) in the sentEvent() function and the broker will send the data to
+the appropriate endpoints.
+
+> TODO:
+> * need to specify what "system information" is, if for no other reason than we
```

Comment:
We get a lot of this system information "for free"  on Telemetry thanks to the [environment data](http://gecko.readthedocs.io/en/latest/toolkit/components/telemetry/telemetry/data/environment.html) added to the ping. I think a lot of that will be inaccessible to library code running in webextension-based experiments. Need to inventory what we can actually gather in that scenario. (Could be more than I think off the top of my head)